src/app/globals.css

```
@@ -1974,3 +1974,1403 @@ html.custom-cursor-active * {
+  left: 0;
+  right: 0;
+  top: 50%;
+  height: 1px;
+  background: rgba(0, 0, 0, 0.6);
+  box-shadow: 0 1px 0 rgba(255, 255, 255, 0.03);
+  transform: translateY(-50%);
+}
+.footer-index__tile.is-flipping {
+  animation: mfFlapTick 0.11s linear infinite;
+}
+@keyframes mfFlapTick {
```

Stylelint (17.14.0)

[error] Expected keyframe name "mfFlapTick" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)